apps/xmtp.chat/src/components/Messages/IntentContent.tsx

```diff
@@ -25,9 +25,12 @@ export const IntentContent: React.FC<IntentContentProps> = ({
     <>
       <DateLabel date={nsToDate(sentAtNs)} align="center" padding="sm" />
       <Group gap="4" wrap="wrap" justify="center">
-        <Identity
+        <IdentityBadge
           address={senderMember ? getMemberAddress(senderMember) : ""}
-          inboxId={senderInboxId}
+          displayName={senderInboxId}
+          tooltip={
+            senderMember ? undefined : "This member is no longer in the group"
```

Comment:
We have the text `"This member is no longer in the group"` quite often. Shall we put it into a strings file?